mysql-db-init/README.md

```diff
@@ -20,6 +20,7 @@ database (schema) initialization helper
 |                      | Description          |
 | :------------------- | :------------------- |
 | **MYSQL_HOST**       | MYSQL host           |
+| **MYSQL_SUPER_USER** | MYSQL admin user     |
```

Comment:
As mentioned on Slack, maybe I'm missing something obvious but I don't see the point of having the superuser available as a variable since root will be the one created by default when the initialization happens. This would only be needed if for whatever reason you just drop the root superuser. Also, if you really did want to keep it like this, you'd also need to change the line with `docker exec -it mysql mysql -h <local-ip> -uroot -p mysql` to `docker exec -it mysql mysql -h <local-ip> -u$MYSQL_SUPER_USER -p mysql`

Reply:
The variable could be set to `root` if empty. I would keep the variable as like this it can use another superuser that is not named `root` even if we have no use case for it currently. Also keep in mind that this repo is open.

Reply:
It could be set to `root` if empty but then we lose in visibility for how the superadmin user account is named and would need to check the script. I think it's best to just leave it as it is and to set the value explicitly like Jasmin did.